MSAL/src/native_auth/network/responses/validator/reset_password/MSALNativeAuthResetPasswordResponseValidator.swift

```diff
@@ -71,9 +71,9 @@ final class MSALNativeAuthResetPasswordResponseValidator: MSALNativeAuthResetPas
         guard let apiError = error as? MSALNativeAuthResetPasswordStartResponseError else {
             MSALLogger.log(level: .error,
                            context: context,
-                           format: "Error type not expected")
+                           format: "Reset Password Start: Error was not decoded properly, error: \(error)")
 
-            return .unexpectedError
+            return .unexpectedError(message: "Unexpected response body received")
```

Comment:
can you please create a new constant in the MSALNativeAuthErrorMessage enum and use it?

Reply:
Done